File: updraft-sdk/src/main/java/com/appswithlove/updraft/api/request/CheckLastVersionRequest.java
package com.appswithlove.updraft.api.request;

import com.google.gson.annotations.SerializedName;

/**
 * Created by satori on 3/27/18.
 */

public class CheckLastVersionRequest {

    @SerializedName("sdk_key")
    private String mSdkKey;

    @SerializedName("app_key")
    private String mAppKey;

    @SerializedName("version")
    private String mVersion;

    public String getSdkKey() {
        return mSdkKey;
    }

    public void setSdkKey(String sdkKey) {
        mSdkKey = sdkKey;
    }

    public String getAppKey() {
        return mAppKey;
    }

    public void setAppKey(String appKey) {
        mAppKey = appKey;
    }

    public String getVersion() {
        return mVersion;
    }

    public void setVersion(String version) {
        mVersion = version;
    }
}

File: updraft-core/src/commonMain/kotlin/com/appswithlove/updraft/api/request/CheckLastVersionRequest.kt
package com.appswithlove.updraft.api.request

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CheckLastVersionRequest(
    @SerialName("sdk_key") val sdkKey: String? = null,
    @SerialName("app_key") val appKey: String? = null,
    @SerialName("version") val version: String? = null,
)
